src/ios/CDVCamera.m

```diff
@@ -670,17 +670,21 @@ - (void)imagePickerControllerReturnImageResult
 {
     CDVPictureOptions* options = self.pickerController.pictureOptions;
     CDVPluginResult* result = nil;
+   
+    NSMutableData *dest_data = [NSMutableData data];
```

Comment:
please use camelCase for variable names. Besides that, can we find a better name than `dest_data`?

Reply:
@timbru31 i renamed it to `imageDataWithExif`. is that fine for you?